packages/fast-development-site-react/webpack.config.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
     },
     module: {
         rules: [
+            {
```

Comment:
it may be better to use somthing like https://github.com/jantimon/favicons-webpack-plugin - it will cover more bases

Reply:
@Falkicon created only a 16x16 favicon ico (not based off of a larger image), so I don't think we can use the above yet.